fe1-web/components/TextInputList.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { View } from 'react-native';
+import RemovableTextInput from './RemovableTextInput';
+
+/**
+ * Component which creates a list of Text input fields which can be deleted individually
+ * Returns array with non-empty Values of the text fields
+ * It uses the RemovableTextInput.tsx component to show the individual inputs
+ * Input: A function that stores the values that were inputted by the user
+ * Output: Array of the unique, nonempty values inputted by the user
+ */
+
+const TextInputList = (props: IPropTypes) => {
+  const { onChange } = props;
+  const [idCount, setIdCount] = useState(1);
+  const [userOptions, setUserOptions] = useState([{ id: 0, value: '' }]);
```

Comment:
`options` is counterintuitive in the context of a `TextInputList` , it's gotten me confused a few times.

I would suggest renaming to `userInputs, setUserInputs` and the functions: `addInput`, `removeInput`, etc.

Reply:
Very good suggestion, this will be updated.